src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts

```diff
@@ -269,6 +287,7 @@ export class AcrossConfigStoreClient extends BaseAbstractClient {
   }
 
   async _update(): Promise<ConfigStoreUpdate> {
+    const chainId = this.chainId ?? (await this.configStore.provider.getNetwork()).chainId;
```

Comment:
Because we consciously use a StaticJsonRpcProvider, this should not result in an external RPC call.

Reply:
can we still Promise.all it with the below call?

Reply:
Yeah, that's also fine and more consistent. I will apply that change.

Reply:
Implemented here: https://github.com/across-protocol/sdk-v2/pull/365/commits/ee39a86be93d0507766ef5c1755a91572ba61e7e